x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/stats.tsx

```diff
@@ -104,17 +105,16 @@ export function MobileStats({
 
   const metrics: MetricDatum[] = [
     {
-      color: euiTheme.eui.euiColorDisabled,
+      color: euiTheme.eui.euiColorLightestShade,
       title: i18n.translate('xpack.apm.mobile.metrics.crash.rate', {
-        defaultMessage: 'Crash Rate (Crash per minute)',
-      }),
-      subtitle: i18n.translate('xpack.apm.mobile.coming.soon', {
```

Comment:
I think these translations need to be revisited. It appears they are based off of `Crash Rate (Crashes per minute)` where this will be displaying % of crashed sessions.